great_expectations/checkpoint/util.py

```diff
@@ -38,6 +39,42 @@ def send_slack_notification(query, slack_webhook):
                     text=response.text,
                 )
             )
+
+        else:
+            return "Slack notification succeeded."
+
+
+def send_slack_app_notification(query, slack_channel, slack_token):
```

Comment:
This function seems repetitive to me. Is there any reason why we can't include `slack_channel` and `slack_token` in the old function `send_slack_notification`? I'm envisioning `slack_webhook`, `slack_channel`, and `slack_token` all as optional parameters and deciding how to use them depends on which are provided.

Reply:
Good call. This dried up the code a lot.